flink-runtime-web/src/test/java/org/apache/flink/runtime/webmonitor/history/HistoryServerTest.java

```diff
@@ -78,6 +78,7 @@
 		.disable(JsonGenerator.Feature.AUTO_CLOSE_JSON_CONTENT);
 	private static final ObjectMapper OBJECT_MAPPER = new ObjectMapper()
 		.enable(DeserializationFeature.FAIL_ON_MISSING_CREATOR_PROPERTIES);
+	private static final String JSON_FILE_ENDING = ".json";
```

Comment:
nit: can we reuse the variable defined in HistoryServerArchiveFetcher.java?

Reply:
I guess that wouldn't be problem. There could be even more constants shared between `HistoryServerArchiveFetcher` and `HistoryServerTest` to make the code a bit cleaner.

Reply:
I made the `JSON_FILE_ENDING` package private to be used in test as well.